src/Evolution/Systems/ForceFree/Characteristics.hpp

```diff
@@ -0,0 +1,53 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Tags {
+template <typename Tag>
+struct Normalized;
+}  // namespace Tags
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace ForceFree {
+namespace Tags {
+/*!
+ * \brief Compute the largest characteristic speed of the GRFFE system with
+ * divergence cleaning.
+ *
+ * Wave speeds of the fast modes of the GRFFE system are the speed of light.
+ *
+ * \f{align*}
+ *  \lambda_\pm = - n_i \beta^i \pm \alpha
```

Comment:
This isn't the largest char speed. We compute the char speeds in the volume, so there's no face normal. See how GH does it.

Reply:
Ok, so this is just for computing the global maximum speed used for defining the time step?

Reply:
Yes, that's right :)